authentication/account/init_tenant.go

```diff
@@ -0,0 +1,161 @@
+package account
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/fabric8-services/fabric8-auth/account/tenant"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/goasupport"
+	"github.com/fabric8-services/fabric8-auth/log"
+	"github.com/fabric8-services/fabric8-auth/rest"
+
+	goaclient "github.com/goadesign/goa/client"
+)
+
+type tenantConfig interface {
```

Comment:
We already have tenant service - https://github.com/fabric8-services/fabric8-auth/blob/master/authentication/account/service/tenant.go
Let's add missing methods there instead of creating a new one.